qe-tests/cypress/integration/tests/vmware/config_separate_mapping.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  LoginData,
+  MappingData,
+  MappingPeer,
+  PlanData,
+  TestData,
+  VmwareProviderData,
+} from '../../types/types';
+import { providerType, storageType } from '../../types/constants';
+const url = Cypress.env('url');
+const user_login = 'kubeadmin';
+const user_password = Cypress.env('pass');
+const v2v_vmware_username = Cypress.env('v2v_vmware_username');
+const v2v_vmware_password = Cypress.env('v2v_vmware_password');
+const v2v_vmware_hostname = Cypress.env('v2v_vmware_hostname');
+const v2v_vmware_cert = Cypress.env('v2v_vmware_cert');
+
+export const loginData: LoginData = {
+  username: user_login,
+  password: user_password,
+  url: url,
+};
+
+export const providerData: VmwareProviderData = {
+  type: providerType.vmware,
+  name: 'qe-vmware',
+  // hostname: '10.8.58.136',
+  // username: 'administrator@vsphere.local',
+  // password: 'Tux4Linux!',
```

Comment:
Can you at least remove the password itself if not all these lines? We shouldn't commit them into history at all if we can help it. If you push a commit removing it, when this PR is squashed and merged, it will no longer be in the history on the main branch.

Reply:
Cleaned this up, now there are no clear text passwords. Each config accepts credentials as CLI parameters.